README.md

```diff
@@ -111,49 +110,58 @@ print(user_details)  # UserDetails(name="John Doe", age=30)
 ```python
 from openai_function_call import OpenAISchema
 from openai_function_call.dsl import ChatCompletion, MultiTask, messages as m
+from openai_function_call.dsl.messages import system as s
 
 # Define a subtask you'd like to extract from then,
-# We'll use MultTask to easily map it to a List[Search] 
+# We'll use MultTask to easily map it to a List[Search]
 # so we can extract more than one
 class Search(OpenAISchema):
     id: int
     query: str
 
 tasks = (
     ChatCompletion(name="Acme Inc Email Segmentation", model="gpt-3.5-turbo-0613")
-    | m.ExpertSystem(task="Segment emails into search queries")
+    | s.Identity(identity="World class state of the art agent") # if no identity is provided, this is the default one
```

Comment:
What do you think?

Should it have the SystemIdentity? Makes it more obvious incase someone does not Alias system. 

Could also overload init so it's not 

SystemIdentity(identity=...) 

If you give me some time I can make these changes myself. Don't want to make you run around. 

--

What we probs also need is to figure out how to structure doc strings so we can build auto docs around this.